rollup.config.js

```diff
@@ -40,7 +40,7 @@ const config = {
         }),
         copy({
             targets: [{
-                src: ['www/js/lib/*dec-wasm.wasm', 'www/js/lib/libzim-asm.js', 'www/js/lib/libzim-wasm.*', 'www/js/lib/webpHeroBundle*',
+                src: ['www/js/lib/*dec-wasm.wasm', 'www/js/lib/library.js', 'www/js/lib/libzim-asm.js', 'www/js/lib/libzim-wasm.*', 'www/js/lib/webpHeroBundle*',
```

Comment:
If you place the file directly in `www`, you won't need this addition. If you decide to keep it in `lib`, then as well as copying it, you will need to do a search-and-replace inside `library.html` to change the path, because we don't keep `lib`.